patterns/hidden-sidebar.php

```diff
@@ -6,64 +6,87 @@
+<div class="wp-block-group">
+
+<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
+<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0"><!-- wp:avatar {"size":80,"style":{"border":{"radius":"16px"}}} /-->
+
+<!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
+<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'About the author', 'twentytwentyfour' ); ?></h2>
 <!-- /wp:heading -->
 
-<!-- wp:post-author-biography {"fontSize":"small"} /-->
-</div>
+<!-- wp:post-author-biography {"fontSize":"small"} /--></div>
 <!-- /wp:group -->
 
 <!-- wp:separator {"backgroundColor":"base-3","className":"is-style-wide"} -->
 <hr class="wp-block-separator has-text-color has-base-3-color has-alpha-channel-opacity has-base-3-background-color has-background is-style-wide"/>
 <!-- /wp:separator -->
 
-<!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
-<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></h2>
-<!-- /wp:heading -->
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+	<!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
+	<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false}} -->
+	<div class="wp-block-query"><!-- wp:post-template -->
+	<!-- wp:group {"style":{"spacing":{"blockGap":"2px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+	<div class="wp-block-group"><!-- wp:post-title {"isLink":true,"fontSize":"medium"} /-->
 
-<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false}} -->
-<div class="wp-block-query"><!-- wp:post-template -->
-<!-- wp:group {"style":{"spacing":{"blockGap":"0.5rem"}},"layout":{"type":"flex","orientation":"vertical"}} -->
-<div class="wp-block-group"><!-- wp:post-title {"isLink":true,"fontSize":"medium"} /-->
+	<!-- wp:group {"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0.3em"}},"layout":{"type":"flex","justifyContent":"left"}} -->
+	<div class="wp-block-group"><!-- wp:post-date {"format":"M j, Y","isLink":true} /-->
 
-<!-- wp:pattern {"slug":"twentytwentyfour/post-meta"} /-->
+	<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast-2"}}}},"textColor":"contrast-2"} -->
```

Comment:
Why can't the meta pattern be re-used?

Reply:
yes, does it need adjusting?

Reply:
We need to use a post meta template part. https://github.com/WordPress/twentytwentyfour/issues/409

Reply:
So to merge this PR this part it stays a pattern for now, but afterwards with a new PR it's updated to a template part?